packages/auth/src/Expo-AuthSession/refreshToken.ts

```diff
@@ -0,0 +1,10 @@
+import { TokenResponse } from "expo-auth-session";
+import { MadAccount } from "../types";
+import { getConfig, getDiscovery } from "./store";
+
+export const refreshToken = async (token: TokenResponse, userData: MadAccount) => {
```

Comment:
userData is apparently never used according to GH actions

Reply:
fixed